fdbrpc/CMakeLists.txt

```diff
@@ -55,8 +52,14 @@ if(NOT WIN32)
   list(APPEND FDBRPC_SRCS libcoroutine/context.c libeio/eio.c)
 endif()
 
-actor_set(FDBRPC_BUILD "${FDBRPC_SRCS}")
+set(FDBRPC_SRCS_ALLOW_ACTOR_WITHOUT_WAIT
+  ActorFuzz.actor.cpp
+  FlowTests.actor.cpp
+  dsltest.actor.cpp)
+
+actor_set(FDBRPC_BUILD "${FDBRPC_SRCS};${FDBRPC_SRCS_ALLOW_ACTOR_WITHOUT_WAIT}")
 add_library(fdbrpc STATIC ${FDBRPC_BUILD})
 actor_compile(fdbrpc "${FDBRPC_SRCS}")
+actor_compile(fdbrpc "${FDBRPC_SRCS_ALLOW_ACTOR_WITHOUT_WAIT}" DISABLE_ACTOR_WITHOUT_WAIT)
```

Comment:
I'd recommend making this `DISABLE_ACTOR_WITHOUT_WAIT_WARNING`, as without the suffix I think it could be interpreted either way. I say this because I did actually read it the opposite way initially (to be more specific, I read it to mean "don't allow actors without wait statements")